app/models/manageiq/providers/openstack/cloud_manager/vm/associate_ip.rb

```diff
@@ -0,0 +1,31 @@
+module ManageIQ::Providers::Openstack::CloudManager::Vm::AssociateIp
+  def validate_associate_address
+    {:available => true, :message => nil}
+  end
+
+  def validate_disassociate_address
+    {:available => true, :message => nil}
```

Comment:
It might be good validate if floating IP is/is not already associated with the Vm.